src/main/radiusd.c

```diff
@@ -621,6 +625,14 @@ int main(int argc, char *argv[])
 	 */
 	state = fr_state_init(NULL);
 
+#ifdef HAVE_SYSTEMD
+	int ret_notif;
```

Comment:
That'll cause some compilers to throw warnings about declarations only being allowed at the top of the function.  Wrap the whole lot in a pair of curly braces.